.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,10 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Request For Comments (RFC)
+    about: When having an idea on how to improve our processes, propose the idea so that the team can provide feedback.
+    url: https://nimble.link/android-templates-rfc
```

Comment:
FYI: meanwhile in flutter_templates https://github.com/nimblehq/flutter_templates/pull/105/files